app/js/views/microphone.jsx

```diff
@@ -27,17 +27,56 @@ export default class Microphone extends React.Component {
     this.click = this.click.bind(this);
   }
 
+  loadAudio() {
+    const AudioContext = window.AudioContext || window.webkitAudioContext;
```

Comment:
Not sure where are you executing this code, but recent Chromes and Firefoxes have got rid of the webkit prefix. You could also load the WebAudio Monkey Patch polyfill if you want to write code without those ugly prefixes: https://github.com/cwilso/AudioContext-MonkeyPatch